xz/plan.ps1

```diff
@@ -1,11 +1,11 @@
 $pkg_name="xz"
 $pkg_origin="core"
-$pkg_version="5.2.4"
+$pkg_version="5.2.3"
```

Comment:
Is there a reason for going to an older version?  Builder will always return the latest version, so this version would be masked by existing 5.2.4 versions.  If there is a specific reason for it, could you please describe the need for the change so we can evaluate how to proceed?

Reply:
Nice spot @smacfarlane.  Will fix this.

Reply:
Fixed now.

Reply:
This is a strange one.  The actual file shows version 5.2.4 even though this change shows something different.  Check it out by viewing the file not the changes.